README.md

```diff
@@ -31,7 +31,8 @@ Within this Repo there are scripts that have been approved through the PHD by th
 5. *Incarcerations* (Outdated): IncarcerationsMonthly; Incarcerations_Ten; Incarcerations_Five
 
 # Attribution and Lineage
```

Comment:
Just noting, do we want any social links for contributors? Websites, LinkedIns, Githubs, etc.?

Reply:
Feel free to resolve if answer is no.